### PR TITLE
mavproxy_console: add periodic warning of autopilot software failure

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: MAVProxy/modules/mavproxy_console.py [hunk 1]
```diff
@@ -349,6 +350,21 @@ def mavlink_packet(self, msg):
...
+                for field_num in range(1, 5):
+                    field = "errors_count%u" % field_num
+                    x = getattr(msg, field, None)
+                    if x is None:
+                        self.say("Failed to get field %s" % field)
+                        self.last_sys_status_errors_announce = now
+                        break
+                    if x != 0:
+                        self.last_sys_status_errors_announce = now
+                        self.say("System failure detected.  Land now.")
```
Comment (tridge) [inline MAVProxy/modules/mavproxy_console.py]: need to make the msg shorter. It takes a long time to announce these things, and meanwhile the pilot doesn't have airspeed etc announcements

File: MAVProxy/modules/mavproxy_console.py [hunk 2]
```diff
@@ -349,6 +350,21 @@ def mavlink_packet(self, msg):
...
             self.last_sys_status_health = msg.onboard_control_sensors_health
 
+            # check for any error bits being set:
+            now = time.time()
+            if now - self.last_sys_status_errors_announce > self.mpstate.settings.sys_status_error_warn_interval:
+                for field_num in range(1, 5):
+                    field = "errors_count%u" % field_num
+                    x = getattr(msg, field, None)
+                    if x is None:
+                        self.say("Failed to get field %s" % field)
```
Comment (tridge) [inline MAVProxy/modules/mavproxy_console.py]: i don't think we should do a say() here

File: MAVProxy/modules/mavproxy_console.py (hunk 1 above)
Comment (peterbarker) [reply MAVProxy/modules/mavproxy_console.py]: Shortened to "Critical failure"

File: MAVProxy/modules/mavproxy_console.py (hunk 2 above)
Comment (peterbarker) [reply MAVProxy/modules/mavproxy_console.py]: Changed to be a writeln instead.

**Pull request merged**